install/kubernetes/cilium/charts/preflight/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+You have successfully installed Cilium {{ .Chart.name }}
```

Comment:
For preflight, I think the notes page might need to be a bit different if there's an option, because preflight is only pulling the cilium images to all the nodes, it's not installing them yet.